packages/generators/utils/module.js

```diff
@@ -10,10 +10,13 @@
 module.exports = _ => {
 	return {
 		test: `${new RegExp(/\.js$/)}`,
-		exclude: "/node_modules/",
+		include: ["path.resolve(__dirname, 'src')"],
```

Comment:
@sokra just wondering. If the entry folder is different, then this would need to change, right?